app/controllers/application_controller.rb

```diff
@@ -3,9 +3,16 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :null_session
   before_action :default_format_json
+  before_filter :authenticate
```

Comment:
[SUG] I think we must stick to `before_action` instead of mixing `_action` and `_filter` in our app.